docs/dbaas/setting-up.md

```diff
@@ -1,9 +1,9 @@
-# DBaaS
+# Setting up DBaaS
 
-To enable and use the Database as a Service (DBaaS) feature in PMM, see [DBaaS](../../get-started/dbaas.html).
+To enable and use the Database as a Service (DBaaS) feature in PMM, see [DBaaS](../dbaas/get-started.html).
 
-You can use [free K8s provided by Percona](https://www.percona.com/blog/private-dbaas-with-free-kubernetes-cluster/) for evaluation. 
-You can also create K8s on AWS using these [instructions](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/). 
+You can use [free K8s provided by Percona](https://www.percona.com/blog/private-dbaas-with-free-kubernetes-cluster/) for evaluation.
+You can also create K8s on AWS using these [instructions](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/).
```

Comment:
```suggestion
For a Kubernetes cluster that doesn't expire you can use our "easy script", you can find the instructions [here](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/).  

In the sections that follow we'll try to outline the steps to create your own Kubernetes cluster in a few popular ways.  
```

Reply:
I'm also not sure the next section makes any sense now...or at least not where it is...perhaps the steps to install minikube should be in the minikube configure section